unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/NftBreadcrumbComponentView.cs

```diff
@@ -34,6 +39,69 @@ public override void Awake()
         }
 
         public override void RefreshControl()
+        {
+            ClearInstances();
+
+            categoriesByIndex = new NftSubCategoryFilterComponentView[model.Path.Length];
+
+            for (var i = 0; i < model.Path.Length; i++)
+            {
+                (string Filter, string Name, string Type, bool Removable) subCategory = model.Path[i];
+                bool isLastItem = i == model.Path.Length - 1;
+                bool isSelected = model.Current == i;
+
+                NftSubCategoryFilterComponentView subCategoryView =
+                    CreateSubCategory(subCategory, isLastItem, isSelected);
+                categoriesByIndex[i] = subCategoryView;
+
+                if (!isLastItem)
+                    CreateSeparator();
+            }
+
+            isLayoutDirty = true;
+        }
+
+        private void Update()
+        {
+            if (!isLayoutDirty) return;
+            isLayoutDirty = false;
+            layoutContainer.ForceUpdateLayout(false);
+        }
+
+        private void CreateSeparator()
+        {
+            GameObject separator = Instantiate(separatorPrefab, container, false);
```

Comment:
Won't it also make sense to pool separators, just like you do for `SubCategory`?

Reply:
Done